src/app.js

```diff
@@ -1,3 +1,5 @@
+import LoginForm from '.views/login-form';
```

Comment:
This should be `./views/login-form` just like we can use `./` in our terminal to refer to the current directory, that's what we can do when loading in modules.